core/services/synchronization/telemetry_ingress_batch_client.go

```diff
@@ -77,7 +77,7 @@ func NewTelemetryIngressBatchClient(url *url.URL, serverPubKeyHex string, ks key
 		serverPubKeyHex:   serverPubKeyHex,
 		globalLogger:      lggr,
 		logging:           logging,
-		lggr:              lggr.Named("TelemetryIngressBatchClient"),
+		lggr:              lggr.Named(fmt.Sprintf("TelemetryIngressBatchClient.%s.%s", network, chainID)),
```

Comment:
Can we chain these calls in order to avoid assuming the separator is `.`?
```suggestion
		lggr:              lggr.Named("TelemetryIngressBatchClient").Named(network).Named(chainID),
```

Reply:
Fixed in https://github.com/smartcontractkit/chainlink/pull/11397/commits/2fecf65d2cd10f583f6aa680e0819e6eb691773f